docs/OIDC_provider_setup.md

```diff
@@ -0,0 +1,143 @@
+![image](https://user-images.githubusercontent.com/93269128/159425323-b0b2d57c-bfed-4887-aebd-7a4e54919d20.png)
+2. You can change the domain name and choose the country you want. Change of country will change the host url. 
+![image](https://user-images.githubusercontent.com/93269128/159425368-8f7b0fa1-9106-4c51-a4c6-eed5e7586da2.png)
+3. Click the “Application” in the Application section to go to the application page. Click “Create Application” to create a new application. 
+![image](https://user-images.githubusercontent.com/93269128/159425549-6be44aa8-b678-4ab9-be96-aa7536760c33.png)
+4. Name your application and select “Single Page Web Applications”. 
+![image](https://user-images.githubusercontent.com/93269128/200610755-2c4f973c-1125-4c39-9663-0bb7996ad7ec.png)
+5. Go to “Settings” to get “Domain”, “Client ID” and “Client Secret”.  We have to use these fields to register an Organization SSO ID Provider in graphistry.	
+![image](https://user-images.githubusercontent.com/93269128/159425498-5d8db17d-418e-4618-a7f8-af0ada9ce6a5.png)
+6. Change the sign-in redirect url in the following format. Change the field in bracket to relative field. For example, this is one of the urls used in development http://localhost:8000/o/admin/sso/oidc/test_admin/login/callback/ 
+If you are using an enterprise plan, this is the callback url for it. http://localhost:8000/g/sso/oidc/Site_wide_SSO_Provider/login/callback/ 
+![image](https://user-images.githubusercontent.com/93269128/159430424-cfae2dbf-2346-4a0f-bfa6-dfda485433ca.png)
+7. Save changes after completing add callback urls.
+8. Disable grants to use organization function. 
+![image](https://user-images.githubusercontent.com/93269128/159425667-1da0d918-d729-41ed-9d8f-05af4e9929a2.png)
+9. Change the Organization setting to “Team members of organizations” and click “Save Changes“. 
+![image](https://user-images.githubusercontent.com/93269128/159425673-2c41e39d-3a46-41da-9428-f0127f60b8f3.png)
+
+
+### AUTH0 USER SETUP
+
+1. Go to ”User Management” and select the user to go to the user page. Click “Create User” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425732-0a9052e1-b1a1-40ad-94af-4d82c3a88110.png)
+2. Fill in all of the fields and click “Create” to create a new user. 
+![image](https://user-images.githubusercontent.com/93269128/159425784-ef15759e-bb20-415d-b712-d65ac689dbe3.png)
+
+### AUTH0 ORGANIZATION SETUP
+
+1. Go to “Organizations” and click “Create Organization” to create a new organization. 	
+![image](https://user-images.githubusercontent.com/93269128/159425859-295b9e3b-1c59-4474-9697-07da4db63371.png)
+2. Name your organization and a name displayed.
+![image](https://user-images.githubusercontent.com/93269128/159425894-3a75a592-df49-459b-a68e-0794aab9dd9e.png)
+3. Please record this “Organization ID” because this ID has to use when create org sso id provider. 	
```

Comment:
Record your “Organization ID” which you'll need when you create your org sso id provider. **